kataude_controller:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 10

  # Position Controllers --------------------------------------
  ShoulderJoint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: ShoulderJoint2
    pid: {p: 5.0, i: 0.01, d: 1.0}
    # pid: {p: 9.0, i: 0.0, d: 0.0}
  xm540_2_position_controller:
    type: effort_controllers/JointPositionController
    joint: xm540_2
    pid: {p: 5.0, i: 0.01, d: 1.0}
  xm540_3_position_controller:
    type: effort_controllers/JointPositionController
    joint: xm540_3
    # pid: {p: 9.0, i: 0.0, d: 0.0}
    pid: {p: 5.0, i: 0.01, d: 1.0}    
  ShoulderJoint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: ShoulderJoint1
    # pid: {p: 9.0, i: 0.0, d: 0.0}
    pid: {p: 5.0, i: 0.01, d: 1.0}    
  ElbowJoint_position_controller:
    type: effort_controllers/JointPositionController
    joint: ElbowJoint
    # pid: {p: 9.0, i: 0.0, d: 0.0}
    pid: {p: 5.0, i: 0.01, d: 1.0}    
  xm540_1_position_controller:
    type: effort_controllers/JointPositionController
    joint: xm540_1
    # pid: {p: 9.0, i: 0.0, d: 0.0}
    pid: {p: 5.0, i: 0.01, d: 1.0}        
  EndEffecter1_position_controller:
    type: effort_controllers/JointPositionController
    joint: EndEffecter1
    pid: {p: 5.0, i: 0.01, d: 1.0}
  Endeffecter2_position_controller:
    type: effort_controllers/JointPositionController
    joint: Endeffecter2
    pid: {p: 5.0, i: 0.01, d: 1.0}
